rnateam.yml

```diff
@@ -27,3 +27,47 @@ tools:
   - name: sshmm
     owner: rnateam
     tool_panel_section_label: RNA Analysis
+
+  - name: cmv
+    owner: rnateam
+    tool_panel_section_label: RNA Analysis 
+    
+  - name: rnalien
+    owner: rnateam
+    tool_panel_section_label: RNA Analysis 
+    
+  - name: fuma
+    owner: yhoogstrate
+    tool_panel_section_label: RNA Analysis 
+    
+  - name: infernal
+    owner: bgruening
+    tool_panel_section_label: RNA Analysis 
+    
+  - name: rnammer
+    owner: lionelguy
```

Comment:
is that true?

Reply:
looks good, but there will be more to add, hopefully soon

Reply:
Same as in workbench: https://github.com/bgruening/galaxy-rna-workbench/blob/0de0d004bc3fc9a22761ed5713abc0c2a480ec81/rna_workbench_2.yml#L85

Reply:
It was updated last 2013, we should take it over or let it die ...

Reply:
Sorry, I'm more strict on EU than on our Docker.

Reply:
is there a reason why we wouldn't take it over? Sorry maybe I missed the point

Reply:
No reason, just time. Make sure a conda-package is there and tests and get it into galaxytools.

Reply:
I removed it for now, please feel free to take it over to galaxytools.